src/app/Login.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState } from 'react';
+
+import styles from './Login.module.css';
+
+interface LoginProps {
+  goLogin: () => void;
+}
+
+const Login: React.FC<LoginProps> = ({ goLogin }) => {
```

Comment:
```suggestion
type LoginProps = {
  goLogin: () => void;
};

const Login = ({ goLogin }: LoginProps) => {
```

취향의 영역일 수 있는데, 개인적으로는 FC를 안 좋아하고, interface 보다는 type 을 선호합니다!

Reply:
확인했습니다!